sonar-project.properties

```diff
@@ -8,6 +8,7 @@ sonar.typescript.lcov.reportPaths=coverage/lcov.info
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
 sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/main/settings/**/*,src/main/server.ts,src/main/assets/**, \
   src/main/lib/**,src/main/public/**
+sonar.cpd.exclusions=src/main/steps/*/content.ts,
```

Comment:
Don't we need to add tests with languageAssertions instead completely excluding them?

Reply:
This excludes the files for duplicated code, it shouldn't exclude them for tests.